saemref/files/saemref-supervisor.conf

```diff
@@ -1,7 +1,7 @@
 {% from "saemref/map.jinja" import saemref with context -%}
 [program:saemref]
 {% if saemref.instance.wsgi -%}
-command=/home/{{ saemref.instance.user }}/venv/bin/gunicorn 'cubicweb.pyramid:wsgi_application()' --threads {{ saemref.instance.wsgi_threads }} --workers {{ saemref.instance.wsgi_workers }} --bind 0.0.0.0:{{ saemref.instance.port }}
+command=/home/{{ saemref.instance.user }}/venv/bin/gunicorn 'cubicweb.pyramid:wsgi_application()' --forwarded-allow-ips * --threads {{ saemref.instance.wsgi_threads }} --workers {{ saemref.instance.wsgi_workers }} --bind 0.0.0.0:{{ saemref.instance.port }}
```

Comment:
I wonder if we shouldn't use a configuration file for gunicorn. This `command` line is getting quite long.